packages/web-components/src/components/card/card.ts

```diff
@@ -155,7 +156,7 @@ class C4DCard extends CTAMixin(StableSelectorMixin(CDSLink)) {
   protected _renderDisabledLink() {
     const { _classes: classes } = this;
     return html`
-      <div id="link" class="${classes}">${this._renderInner()}</div>
+      <div id="link" class="${classes}" part="link">${this._renderInner()}</div>
```

Comment:
It concerns me that this is getting the same part name as the `<a>` added in the `render` method. Can you change this to something like `part="disabled-link"`?

Reply:
Hi @jkaeser! 
Thanks for the review!!! 
Addressed in my latest commit!